CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus v11.0.10 (February 15, 2024) ###
+
+* Removes pg_send_cancellation and all references (#7509)
```

Comment:
As we discussed in older PRs, please refer to actual PRs making changes on main:

```suggestion
* Removes pg_send_cancellation and all references (#7135)
```